src/styles/main.scss

```
@@ -10,5 +10,6 @@ body {
   box-sizing: border-box;
 }
 
+@import "_variables.scss";
```

I guess, the following line would have the same effect but, as compared, looks simpler
```suggestion
@import "./variables";
```